### ISSUE TITLE
Adding LeetCode Challenge: 24

### ISSUE THREAD

**Issue by mjoyharris** [OpenGenus/cosmos#6369]
**This is a(n):**

- [x] New algorithm
- [ ] Update to an existing algorithm
- [ ] Error
- [ ] Proposal to the Repository

**Details:**

Hi! I wanted to add the common LeetCode challenge called "24 Game."

**Comment by eddiep12** [comment]
Hi, is it okay if I create a folder called LeetCode?

**Comment by mjoyharris** [comment]
Of course! Best,Maya HarrisOn Oct 3, 2024, at 2:46 PM, eddiep12 ***@***.***> wrote:﻿
Hi, is it okay if I create a folder called LeetCode?

**Comment by eddiep12** [comment]
https://github.com/OpenGenus/cosmos/pull/6807
